data-reporting-service/post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -37,7 +39,12 @@ public class PostProcessingService {
     private final PageBuilderRepository pageBuilderRepository;
 
     static final String PAYLOAD = "payload";
-    static final String SP_EXECUTION_COMPLETED = "Stored proc execution completed.";
+    static final String INVESTIGATION = "investigation";
```

Comment:
Do you think we can make it as an `enum`

Reply:
@alog-enquizit yes, I even tried an implementation, but there is a problem with passing the repositories, in a construction like:

```
enum TopicProcessor {
    ORGANIZATION(ORGANIZATION, "sp_nrt_organization_postprocessing") {
        @Override
        void process(String keyTopic, List<Long> ids) {
            processTopic(keyTopic, ids, organizationRepository::executeStoredProcForOrganizationIds, this.topic, this.storedProc);
        }
    },
    PROVIDER(PROVIDER, "sp_nrt_provider_postprocessing") {
        @Override
        void process(String keyTopic, List<Long> ids) {
            processTopic(keyTopic, ids, providerRepository::executeStoredProcForProviderIds, this.topic, this.storedProc);
        }
    },
```
but repos and their methods are not static... I will think a bit more in the next implementation.